packages/lumx-core/src/scss/components/dialog/_index.scss

```diff
@@ -51,22 +55,39 @@
         display: flex;
         flex-direction: column;
         transform: translateY(0);
-        animation: lumx-dialog-show $lumx-dialog-transition-duration;
+
+        #{$self}--is-hidden & {
+            @include mq($until: desktop) {
+                transform: translateY(100%);
+            }
+        }
+
+        @include mq($until: desktop) {
+            transition: transform 500ms cubic-bezier(0.23, 1, 0.32, 1);
+            animation: lumx-dialog-show-mobile 500ms cubic-bezier(0.23, 1, 0.32, 1);
```

Comment:
Could we store the duration in a variable `$lumx-responsive-dialog-transition-duration`, it is useful to synchronize other animations with dialog opening/closing. (`$lumx-dialog-transition-duration` is used in Lumapps)